lib/moonshot/build_mechanism/travis_deploy.rb

```diff
@@ -79,9 +85,24 @@ def wait_for_build(version)
     end
 
     def wait_for_job(job_number)
-      cmd = "bundle exec travis logs #{@cli_args} #{job_number}"
-      # This log tailing fails at the end of the file. travis bug.
-      sh_step(cmd, fail: false)
+      # Find the travis job.
```

Comment:
Could this block be a helper?  At least, through line 91, to enable its reuse in other methods.